integration-tests/api/__tests__/admin/order.js

```diff
@@ -620,6 +636,47 @@ describe("/admin/orders", () => {
         }),
       ]);
     });
+
+    it("Only allow canceling claim after canceling return and fulfillment", async () => {
+      const order_id = "order-with-claim";
+
+      const order = await callGet({
+        path: `/admin/orders/${order_id}`,
+        get: "order",
+      });
+
+      const claim = order.claims[0];
+      const claim_id = claim.id;
+
+      const expectCancelToReturn = partial(expectPostCallToReturn, {
```

Comment:
Here we simply create a partial call, so that we can reuse it to try to cancel the order with different codes.